net/zerotier/patches/0007-fix-system-include-path.patch

```diff
@@ -0,0 +1,25 @@
+From ce428394e93b8bf26fe3e87d7edd92002938bcf6 Mon Sep 17 00:00:00 2001
+From: Liangbin Lian <jjm2473@gmail.com>
+Date: Tue, 10 Jun 2025 10:53:13 +0800
+Subject: [PATCH] fix system include path
+
+`-isystem ext` makes `#include <miniupnpc/miniupnpc.h>` actually include `ext/miniupnpc/miniupnpc.h`.
+
+we should use `$(STAGING_DIR)/usr/include` as a higher priority system include path.
+
+Signed-off-by: Liangbin Lian <jjm2473@gmail.com>
+---
+ make-linux.mk | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+--- a/make-linux.mk
++++ b/make-linux.mk
+@@ -9,7 +9,7 @@ ifeq ($(origin CXX),default)
+ 	CXX:=$(shell if [ -e /opt/rh/devtoolset-8/root/usr/bin/g++ ]; then echo /opt/rh/devtoolset-8/root/usr/bin/g++; else echo $(CXX); fi)
+ endif
+ 
+-INCLUDES?=-Irustybits/target -isystem ext -Iext/prometheus-cpp-lite-1.0/core/include -Iext-prometheus-cpp-lite-1.0/3rdparty/http-client-lite/include -Iext/prometheus-cpp-lite-1.0/simpleapi/include
++INCLUDES?=-Irustybits/target -isystem $(STAGING_DIR)/usr/include -isystem ext -Iext/prometheus-cpp-lite-1.0/core/include -Iext-prometheus-cpp-lite-1.0/3rdparty/http-client-lite/include -Iext/prometheus-cpp-lite-1.0/simpleapi/include
```

Comment:
Isnt there some better way to do it? Have you reached the upstream developers to know about our problem and what they can do for us? Right now, the thing is that you are adding something to OpenWrt repositories, which will stay here forever and we will keep refreshing it for ages.